packages/cookie-manager/src/types.ts

```diff
@@ -41,7 +41,9 @@ export enum TrackerType {
 export type Tracker = {
   id: string;
   type: TrackerType;
+  sessionCookie?: boolean;
```

Comment:
Please make sure to call out in the docs any triggers that will result in these epherermal cookies being dropped.  We want the use to know if it's their responsiblity to delete them or if cookie manger is and if so what triggers that delete.